src/Microsoft.PowerShell.AccountManagement/commands/NewUserCommand.cs

```diff
@@ -0,0 +1,401 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Management.Automation;
+using System.DirectoryServices.AccountManagement;
+
+namespace Microsoft.PowerShell.Commands
+{
+    /// <summary>
+    /// The New-LocalUser cmdlet creates a new local user account.
+    /// </summary>
+    [Cmdlet(VerbsCommon.New, "User",
+            DefaultParameterSetName = "Password",
+            SupportsShouldProcess = true,
+            HelpUri = "https://go.microsoft.com/fwlink/?LinkId=717981")]
+    public class NewUserCommand : PSCmdlet
+    {
+        /// <summary>
+        /// Gets or sets a DateTime that specifies the date and time that the account expires.
+        /// </summary>
+        [Parameter(ValueFromPipelineByPropertyName = true)]
+        [AllowNull]
```

Comment:
Null here means never expires?  But you have `-AccountNeverExpires` below?

Reply:
It doesn't work. Will remove.